Component/LinearSolver/Direct/src/sofa/component/linearsolver/direct/SparseCholeskySolver.inl

```diff
@@ -30,7 +31,11 @@ template<class TMatrix, class TVector>
 SparseCholeskySolver<TMatrix,TVector>::SparseCholeskySolver()
     : f_verbose( initData(&f_verbose,false,"verbose","Dump system state at each iteration") )
     , S(nullptr), N(nullptr)
+    , d_type_perm(initData(&d_type_perm, "permutation", "Type of fill reducing permutation"))
+    , d_type_permOptions(3,"None", "SuiteSparse", "METIS")
```

Comment:
There is no need to have 2 variables for the same thing here: d_type_perm contains the options group and the selected value, so there is no need for `d_type_permOptions`